lib/segment/src/types.rs

```diff
@@ -355,6 +355,7 @@ pub struct ScalarQuantizationConfig {
     pub r#type: ScalarType,
     /// Quantile for quantization. Expected value range in (0, 1.0]. If not set - use the whole range of values
     #[serde(skip_serializing_if = "Option::is_none")]
+    // TODO: validation should exclude 0.0
```

Comment:
I guess we can set min value as 0.5. Usual quantile interval - 0.95-1.0

Reply:
Will do that now!